package.json

```diff
@@ -13,6 +13,8 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "concurrently": "^7.1.0"
+    "concurrently": "^7.1.0",
+    "styled-component": "^2.8.0",
+    "styled-components": "^5.3.5"
```

Comment:
You installed here `styled-components` into your `root`  `node_modules`. This makes the project unnecessarily big. Just uninstall it with

```sh
npm uninstall <package-name>
```
Read more[ here](https://flaviocopes.com/npm-uninstall-packages/) if you like.

Reply:
Done!